go.mod

```diff
@@ -11,14 +11,18 @@ require (
 	github.com/xlab/treeprint v1.2.0
 )
 
+require (
+	github.com/moby/sys/user v0.1.0 // indirect
+	github.com/rivo/uniseg v0.4.4 // indirect
+)
+
 require (
 	github.com/docker/go-units v0.5.0 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/klauspost/compress v1.17.7 // indirect
 	github.com/klauspost/pgzip v1.2.6 // indirect
-	github.com/mattn/go-runewidth v0.0.9 // indirect
+	github.com/mattn/go-runewidth v0.0.15 // indirect
```

Comment:
@adrianreber actually my PR updated this dependency and I think this is required by table writer and thus this dependency pulling the `rivo` .

Reply:
This should not be part of this PR. Please undo it.

Reply:
yes sure